docs/src/components/TrustTriangleDiagram.css

```
@@ -19,50 +19,78 @@
+
+.tt-diagram__icon--issuer  * { stroke: #d97706; }
+.tt-diagram__icon--holder  * { stroke: #2563eb; }
+.tt-diagram__icon--verifier* { stroke: #7c3aed; }
+
+[data-theme='dark'] .tt-diagram__icon--issuer  * { stroke: #fbbf24; }
+[data-theme='dark'] .tt-diagram__icon--holder  * { stroke: #60a5fa; }
+[data-theme='dark'] .tt-diagram__icon--verifier* { stroke: #a78bfa; }
+
+/* ── Edges ───────────────────────────────────────────────────── */
 .tt-diagram__edges line {
   stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)